mscore/shortcut.cpp

```diff
@@ -3879,7 +3879,7 @@ void Shortcut::retranslate()
 
 void Shortcut::refreshIcons()
       {
-      foreach (Shortcut* s, _shortcuts) {
+      for (Shortcut* s: _shortcuts) {
```

Comment:
minor nitpick: we prefer the : to be surrounded by spaces: `for (Shortcut* s : _shortcuts) {`